pkg/storage/kubernetes/ipam.go

```diff
@@ -35,6 +35,9 @@ func NewKubernetesIPAM(containerID string, ipamConf whereaboutstypes.IPAMConfig)
 		return nil, err
 	} else if ctx, ok := cfg.Contexts[cfg.CurrentContext]; ok && ctx != nil {
 		namespace = ctx.Namespace
+		if namespace == "" {
+			namespace = "kube-system"
```

Comment:
I think you should use the `metav1.NamespaceSystem` constant instead of this hard-coded string. 

I would say I also miss a unit test for this, but the whole pkg as none .... Not good.